github3/issues/issue.py

```diff
@@ -240,6 +242,15 @@ def is_closed(self):
             return True
         return False
 
+    def is_locked(self):
+        """Checks if the issue is locked.
+
+        :returns: bool
+        """
+        if self.locked:
```

Comment:
I think this can be simplified. Since self.locked will return `True` or `False`, you can do something like:

`return self.locked`